Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,269 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Globalization;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace BootCamp.Chapter
 {
     public static class BalanceStats
     {
+        public static string[] allNames { get; set; }
+        public static float[][] allBalances { get; set; }
+        public static void ParsePeopleAndBalance(string[] peopleAndBalances)
+        {
+            if(peopleAndBalances[0] == "") { return; }
```

Comment:
What if it's null? Unlikely, but it's standard to check empty/null strings with `string.IsNullOrEmpty(input)`